Scripts/AreaLightTest.bv.lua

```diff
@@ -109,7 +111,7 @@ function VaryMetallicOnly()
     SetImguiValue('AreaLightSample/Min Max Metallic', Vector2(0.0, 1.0))
 end
 
-g_screenshotOutputFolder = ResolvePath('@user@/Scripts/Screenshots/AreaLights/')
+g_screenshotOutputFolder = ResolvePath('@user@/Scripts/Screenshots/' .. g_testEnv .. '/AreaLights/')
```

Comment:
We could put a utility function like this in TestEnvironment.lauc:
```
function MakeScreenshotPath(testCaseName, testStepName)
    return '@user@/Scripts/Screenshots/' .. g_testEnv .. '/' .. testCaseName .. '/' .. testStepName .. '.png'
end
```
Then remove all of the g_screenshotOutputFolder variables. And wherever CaptureScreenshot is called, do it like this:
```
CaptureScreenshot(MakeScreenshotPath("AreaLightTest", "WhateverTestStepName"))
```